### ISSUE TITLE
Opening file/txt file

### ISSUE THREAD

**Issue by mazematt** [thebaselab/codeapp#766]
I’m learning phyton currently, im using code line with open (‘x.txt’) as data. I created the txt file called x in the same folder as my python project just as I would on a pc. But code app does not seem to find it. 

This is the error: 
File "/private/var/mobile/Containers/Data/Application/3BC03145-FCE6-4F09-BBF5-906E09193DAF/Documents/default.py", line 1
    with open ('x.txt') as adat

Can you help me solve the problem? 
Or did I do something wrong?
I did not find any help or hints on how to open, read and write files in the code app.

Thank you for your help!
                               ^

**Comment by bummoblizard** [comment]
It’s likely that you are in a wrong working directory in the terminal. For me to take a better look at this, please upload a screenshot with the file directory tree,  terminal and the source code shown.

**Comment by mazematt** [comment]
![126A51A2-9962-4710-82B5-B65D89C4A714](https://user-images.githubusercontent.com/124900603/217846549-a4b5a863-798e-4243-84a5-f8fe64ed4bc7.png)
![778D30F1-FE26-43BE-8BB7-A8845A768122](https://user-images.githubusercontent.com/124900603/217846570-8e636e3c-6865-4742-ad39-0d20883d0f65.png)

**Comment by mazematt** [comment]
These are the two files I used, I’m just getting used to phyton

**Comment by bummoblizard** [comment]
Can you also show me the complete error message?

**Comment by mazematt** [comment]
![00088553-89ED-4342-9C0C-952B42074240](https://user-images.githubusercontent.com/124900603/217936774-8cf31d7f-9b6c-4c8c-a0b0-ad863aa91be5.png)
![845D4D15-F629-4241-961C-1242B391887C](https://user-images.githubusercontent.com/124900603/217936782-861d4328-3913-4546-8275-efe893d458e3.png)

**Comment by bummoblizard** [comment]
It worked in the first picture you sent. Can you try to run the code directly in the terminal by typing `python3 default.py` and see if it works?

You might also try to restart your device.

**Comment by TonyHbox** [comment]
Hello,
I’m having the same problem, and I tried the solutions suggested above with no success. Like the original poster I’m new to Python (but not coding), and writing on my iPad. I’ve tried opening a file (see pics below) using the relative path and the absolute path, or what Ctrl-C from the Files app copied, but no success. My iPad has the latest iOS 16.6.1. 

In the following screenshot I tried first to run the program directly from the terminal, and the second time I clicked the play button at the top right. Both gave the same error message, File Not Found, but different reasons.
![IMG_0539](https://github.com/thebaselab/codeapp/assets/145166264/731f9865-2c80-427f-b638-3d101d27d276)
